console-subscriber/src/builder.rs

```diff
@@ -56,6 +56,10 @@ pub struct Builder {
     /// Any scheduled times exceeding this duration will be clamped to this
     /// value. Higher values will result in more memory usage.
     pub(super) scheduled_duration_max: Duration,
+
+    #[cfg(feature = "grpc-web")]
+    /// Cors layer for grpc-web.
+    cors_layer: Option<tower_http::cors::CorsLayer>,
```

Comment:
As described below, I think this shouldn't be part of `console-subscriber`.

Reply:
Removed.